IDEAS/Examples/PPD12/BaseClasses/HeatingNoControl.mo

```diff
@@ -13,15 +13,21 @@ partial model HeatingNoControl "Ppd 12 example model without control"
 
   //CONTROL
   parameter Modelica.SIunits.Temperature TSet=294.15 "Temperature set point";
-  parameter Modelica.SIunits.Temperature TSet2=296.15 "Temperature set point";
+  parameter Modelica.SIunits.Temperature TSet2=303.15 "Temperature set point";
+
+  Modelica.SIunits.Efficiency eta = {-6.017763e-11,2.130271e-8,-3.058709e-6,2.266453e-4,-9.048470e-3,1.805752e-1,-4.540036e-1}*{TCorr^(6-i) for i in 0:6};
```

Comment:
efficiencies of?

Reply:
The gas boiler? It would be useful to have a comment.

Reply:
missed that, thanks!